Add AWS Security Lake alert visualization in Wazuh dashboard (#6838)
wazuh/wazuh-documentation: source/cloud-security/amazon/services/supported-services/security-lake.rst

After setting the required parameters, restart the Wazuh manager to apply the changes:

.. include:: /_templates/common/restart_manager.rst

.. note::

>     The module's time of execution varies depending on the number of notifications present in the queue. This directly influences the time you have to wait to see every alert on the Wazuh dashboard. If the ``<interval>`` value is less than the required time of execution, the :ref:`Interval overtaken<interval_overtaken_message>` message will be displayed in the ``ossec.log`` file.

Reviewer: ```suggestion

   The module execution time varies depending on the number of notifications in the queue. This affects the time to display alerts on the Wazuh dashboard. If the ``<interval>`` value is less than the execution time, the :ref:`Interval overtaken <interval_overtaken_message>` message appears in the ``ossec.log`` file.
```

More information about the different authentication methods can be found in the :ref:`Configuring AWS credentials <amazon_credentials>` documentation.

Visualizing alerts in Wazuh dashboard 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the ``Security events`` module and, for better visualization, apply the filter ``rule.groups: amazon_security_lake``:

Reviewer 2: We should use bold for user interface elements

```suggestion
Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the **Security events** module and, for better visualization, apply the filter ``rule.groups: amazon_security_lake``:
```

> Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the **Security events** module and, for better visualization, apply the filter ``rule.groups: amazon_security_lake``:

Reviewer: ```suggestion
Once you set the configuration and restart the manager, you can visualize the Amazon Security Lake alerts in the Wazuh dashboard. To do this, go to the **Security events** module. Apply the filter ``rule.groups: amazon_security_lake`` for an easier visualization.
```
